resolwe_bio/management/commands/generate_samples.py

```diff
@@ -0,0 +1,248 @@
+                                  'experiment_type': 'RNA-Seq',
+                                  'molecule': 'polyA RNA',
+                                  'optional_char': ['Cell line: EOC 2'],
+                                  'description': "Mouse brain study. Expression profiling."},
+                          'protocols': {
+                            'fragmentation_method': 'sonication',
+                            'growth_protocol': growth_protocol,
+                            'library_prep': library_prep}}
+        return annotation
+
+    def create_data(self, reads_name='seq_reads', annotated=False):
+        # get test data paths
+        data_path = settings.FLOW_EXECUTOR['DATA_PATH']
+        path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+        reads = os.path.join(path, 'tests', 'files', reads_name + '.fastq.gz')
+        fastqc = os.path.join(path, 'tests', 'files', reads_name + '_fastqc.zip')
+        bam_mapping = os.path.join(path, 'tests', 'files', 'alignment_position_sorted.bam')
+        bai = os.path.join(path, 'tests', 'files', 'alignment_position_sorted.bam.bai')
+        dicty_genes = os.path.join(path, 'tests', 'files', 'dicty_genes.tab.gz')
+        human_genes = os.path.join(path, 'tests', 'files', 'human_genes.tab.gz')
+        mouse_genes = os.path.join(path, 'tests', 'files', 'mouse_genes.tab.gz')
+
+        # crate reads data object
+        d = Data.objects.create(
+            name = self.set_name(),
+            descriptor_schema = DescriptorSchema.objects.get(name='Reads annotation'),
+            descriptor = self.set_reads_descriptor(),
+            status = 'OK',
+            process = Process.objects.get(slug='import-upload-reads-fastq'),
+            contributor = User.objects.get(username='admin'),
+            input = {'src': {'file': os.path.basename(reads)}},
+            output = {
+                'fastq': {'file': os.path.basename(reads)},
+                'fastqc_url': {
+                    'url': 'fastqc/{}_fastqc/fastqc_report.html'.format(reads_name),
+                    'refs': ['fastqc/{}_fastqc'.format(reads_name)], 'name': 'View'},
+                'fastqc_archive': {'file': '{}_fastqc.zip'.format(reads_name)},
+                'number': 13,
+                'bases': "101"})
+
+        # Create data folder and copy reads files
+        os.mkdir(os.path.join(data_path, str(d.id)))
+        shutil.copy(reads, os.path.join(data_path, str(d.id)))
+
+        # Attach FastQC data to reads file
+        os.mkdir(os.path.join(data_path, str(d.id), 'fastqc'))
+        shutil.copy(fastqc, os.path.join(data_path, str(d.id)))
+        
+        with zipfile.ZipFile(fastqc) as zf:
+            zf.extractall(os.path.join(data_path, str(d.id), 'fastqc'))
+
+        shutil.copy(os.path.join(data_path, str(d.id), 'fastqc', reads_name + '_fastqc', 'fastqc_report.html'), os.path.join(data_path, str(d.id), 'fastqc'))
```

Comment:
You can probably rename file with specifying filename in `dst` attribute. Also split this to multiple lines.
 ```
old_fastqc_path = os.path.join(data_path, str(d.id), 'fastqc', reads_name + '_fastqc', 'fastqc_report.html')
new_fastqc_path = os.path.join(data_path, str(d.id), 'fastqc', reads_name + '_fastqc.html')
shutil.copy(old_fastqc_path, new_fastqc_path)
```